osu.Game.Tournament/Components/TournamentBeatmapPanel.cs

```diff
@@ -124,21 +126,11 @@ private void load(LadderInfo ladder, TextureStore textures)
 
             if (!string.IsNullOrEmpty(mods))
             {
-                AddInternal(new Container
+                AddInternal(new ModSprite
                 {
-                    RelativeSizeAxes = Axes.Y,
-                    Width = 60,
                     Anchor = Anchor.CentreRight,
                     Origin = Anchor.CentreRight,
-                    Margin = new MarginPadding(10),
-                    Child = new Sprite
-                    {
-                        FillMode = FillMode.Fit,
-                        RelativeSizeAxes = Axes.Both,
-                        Anchor = Anchor.CentreRight,
-                        Origin = Anchor.CentreRight,
-                        Texture = textures.Get($"mods/{mods}"),
-                    }
+                    Mod = mods
```

Comment:
I'm quite confused by this set, is `mods` always meant to be one mod? can't this be renamed to `mod` then?

Reply:
Yeah it was always one mod but it was always called `mods` afaik.  I should rename it, considering it will still be one mod until we can support multiple mods in a panel. 

https://github.com/ppy/osu/blob/f2236312a0323d8a68117c60ae3e46400e75c27e/osu.Game.Tournament/Components/TournamentBeatmapPanel.cs#L140

Reply:
Change `ModDisplay` to `ModIcon` while at it, simplifies code.